invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/config.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""{{ cookiecutter.site_name }} base Invenio configuration."""
+
+from __future__ import absolute_import, print_function
+
+
+# Identity function for string extraction
+def _(x):
+    return x
+
+# Default language and timezone
+BABEL_DEFAULT_LANGUAGE = 'en'
+BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
+I18N_LANGUAGES = [
+]
+
+BASE_TEMPLATE = "invenio_theme/page.html"
+
+SECRET_KEY = "sdfghsdfghfdghdfghjdfhjdfghjjhg"
```

Comment:
The secret key needs to be generated - i.e. here use:

```
SECRET_KEY = "{{cookiecutter.secret_key }}"
```

and in ``cmd.py`` and generation of a secret key (see https://github.com/inveniosoftware/invenio-ext/blob/master/invenio_ext/script/__init__.py#L79-L88 for how to generate a secret key)